pkgs/by-name/su/surfer/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  fetchFromGitLab,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  wayland,
+  libxkbcommon,
+  libGL,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "surfer";
+  version = "v0.2.0";
```

Comment:
```suggestion
  version = "0.2.0";
```

from the [contributing guidelines](https://github.com/NixOS/nixpkgs/tree/665394853091a08d679eb0c166900cb4f57a5bf5/pkgs#package-naming)

> There are a few naming guidelines:
>
> - The pname attribute should be identical to the upstream package name.
>
> - The pname and the version attribute must not contain uppercase letters — e.g., "mplayer" instead of "MPlayer".
>
> - ***The version attribute must start with a digit e.g., "0.3.1rc2".***

Reply:
Remember to edit commit message accordingly